lib/build.js

```diff
@@ -18,20 +16,28 @@ function build(options) {
             return stat.isDirectory();
         });
 
+    directories.forEach( (dir) => {
+      fs.mkdirs(`${dst}/${dir}`);
+
+      let srcTestFile = `${src}/${dir}/test.json`;
+      let destTestFile = `${dst}/${dir}/test.json`;
+
+      if (fs.existsSync(srcTestFile)) {
+        return fs.copyFileSync(srcTestFile, destTestFile);
+      }
+    });
+
     directories = mapcat(directories, (dir) => [
         `${dir}/web`,
         `${dir}/app`,
         `${dir}/amp`
     ].filter(dir => fs.existsSync(`${src}/${dir}`)));
 
+    directories.forEach( (dir) => {
+      fs.mkdirs(`${dst}/${dir}`);
+    });
+
     return Promise.all(mapcat(directories, (dir) => {
-        let prefix = '';
```

Comment:
This is/was basically a custom implementation of `mkdir -p`, which I think we can forgo in favour of `fs-extra`.